### PR TITLE
[WEEK11]_전서윤

### PR DESCRIPTION
**Pull request by wlrmadbs**
**강의 요약**
[[WEEK11]_전서윤.pdf](https://github.com/Ewha-Euron/6th-Intermediate/files/15365627/WEEK11._.pdf)

**캐글 필사 Pytorch drop out, batch normalize snippet**
https://colab.research.google.com/drive/1KXmQsdaOc8VDi_ywXijylc5oWcMPFvCM?usp=sharing

**깃허브 코드 필사: Batch Normalization** 
https://colab.research.google.com/drive/1kflh8_wFR-tMfkaE2Mv0t5Mk3dmPoK46?usp=sharing

### CODE REVIEW

**Comment by bsoiyou** [comment]
11주차도 고생하셨습니다~!😊
